client/payment-details/dispute-details/dispute-summary-row.tsx

```diff
@@ -0,0 +1,131 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import moment from 'moment';
+import { __, _n, sprintf } from '@wordpress/i18n';
+import { dateI18n } from '@wordpress/date';
+
+/**
+ * Internal dependencies
+ */
+import type { Dispute } from 'wcpay/types/disputes';
+import { HorizontalList } from 'wcpay/components/horizontal-list';
+import { formatCurrency } from 'wcpay/utils/currency';
+import { reasons } from 'wcpay/disputes/strings';
+import { formatStringValue } from 'wcpay/utils';
+import { isAwaitingResponse } from 'wcpay/disputes/utils';
+import { ClickTooltip } from 'wcpay/components/tooltip';
+import Paragraphs from 'wcpay/components/paragraphs';
+import HelpOutlineIcon from 'gridicons/dist/help-outline';
+
+interface Props {
+	dispute: Dispute;
+	daysRemaining: number;
+}
+
+const DisputeSummaryRow: React.FC< Props > = ( { dispute, daysRemaining } ) => {
+	const respondByDate = dispute.evidence_details?.due_by
+		? dateI18n(
+				'M j, Y, g:ia',
+				moment( dispute.evidence_details?.due_by * 1000 ).toISOString()
+		  )
+		: '–';
+
+	const disputeReason = formatStringValue(
+		reasons[ dispute.reason ]?.display || dispute.reason
+	);
+	const disputeReasonSummary = reasons[ dispute.reason ]?.summary || [];
```

Comment:
Using the summary seems like the best option from the strings file without adding new content.